eve/main.yml

```diff
@@ -69,3 +70,18 @@ stages:
     - ShellCommand:
         name: 'Build doc'
         command: tox --workdir /tmp/tox -e docs -- html latexpdf
+
+  pep8:
+    worker:
+      type: kube_pod
+      path: eve/workers/pep8.yaml
+      images:
+        pep8-checker:
+          context: '.'
+          dockerfile: eve/images/pep8-checker/Dockerfile
+    steps:
+    - Git: *git_pull
+    - ShellCommand:
+        name: 'Check syntax'
+        haltOnFailure: true
```

Comment:
Would this be required on the doc stage as well?

Reply:
May be. I'll open a PR for this change